app/views/CyaCreateProfileView.scala.html

```diff
@@ -23,20 +23,20 @@
 
 @(list: SummaryList)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = titleNoForm(messages("checkYourAnswers.title"))) {
+@layout(pageTitle = titleNoForm(messages("cyaCreateRecord.title"))) {
 
-    <h1 class="govuk-heading-l">@messages("checkYourAnswers.h1")</h1>
-    <h2 class="govuk-heading-m">@messages("checkYourAnswers.h2")</h2>
+    <h1 class="govuk-heading-l">@messages("cyaCreateRecord.h1")</h1>
+    <h2 class="govuk-heading-m">@messages("cyaCreateRecord.h2")</h2>
     @govukSummaryList(list)
     <div class="govuk-warning-text">
         <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
         <strong class="govuk-warning-text__text">
-            <span class="govuk-visually-hidden">@messages("checkYourAnswers.warning")</span>
-            @messages("checkYourAnswers.warningText")
+            <span class="govuk-visually-hidden">@messages("cyaCreateRecord.warning")</span>
+            @messages("cyaCreateRecord.warningText")
```

Comment:
All `cyaCreateRecord` should replace with `cyaCreateProfile`

Reply:
fixed